modules/bibdocfile/lib/bibdocfile.py

```diff
@@ -1657,7 +1657,7 @@ def create_new_document(doc_type="Main", rec_links=None):
         except Exception, e:
             run_sql('DELETE FROM bibdoc WHERE id=%s', (doc_id, ))
             register_exception(alert_admin=True)
-            raise InvenioBibDocFileError, e
```

Comment:
How about putting a comment saying that we want this part to explicitly fail and give an error to the user ?